src/shared/loop-util.c

```diff
@@ -452,11 +470,16 @@ static int loop_device_make_internal(
                         if (r < 0 && r != -EOPNOTSUPP)
                                 return r;
 
+                        int copy_dissect_fd = loop_device_fd_dup_for_dissect(copy);
+                        if (copy_dissect_fd < 0)
+                                return copy_dissect_fd;
```

Comment:
`dissect_fd` may be leaked when `new()` below fails.

Reply:
ok